netsim/modules/routing.py

```diff
@@ -715,6 +715,37 @@ def process_static_route_includes(
 def import_static_routes(idx: int,o_name: str,node: Box,topology: Box) -> typing.Optional[list]:
   return None
 
+"""
+Create next-hop information from neighbor addresses and static route data
+"""
+def create_nexthop_data(sr_data: Box,ngb_addr: Box, intf: Box) -> Box:
+  nh_data = data.get_empty_box()
+
+  for af in log.AF_LIST:
+    if af not in ngb_addr:
+      continue
+    if isinstance(ngb_addr[af],str):
```

Comment:
Shouldn't we check for neighbors being in the same VRF as the next hop entry?

Reply:
VRFs have local significance.